### PR TITLE
Change encryption key type

### PR DESCRIPTION
**Pull request by SaiPrasannaGopularam**
# Changes
- Update `encryption_key` data type to be a list to accept more than one kms keys `payments` repo

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -327,5 +327,6 @@ variable "secret_arns" {
 
 variable "encryption_key" {
```
Comment (automationforthepeople) [inline variables.tf:328]:
```suggestion
variable "encryption_keys" {
```

File: variables.tf [hunk 2]
```diff
@@ -327,5 +327,6 @@ variable "secret_arns" {
...
   description = "Kms key to decrypt secrets"
+  type        = list
```
Comment (automationforthepeople) [inline variables.tf:330]:
```suggestion
  type        = list(string)
```

File: variables.tf [hunk 3]
```diff
@@ -327,5 +327,6 @@ variable "secret_arns" {
...
   default     = null
```
Comment (automationforthepeople) [inline variables.tf:331]:
```suggestion
  default     = []
```

Comment (automationforthepeople) [inline variables.tf:331]: shouls this be null or `[]` now that it's a list?

**Review by automationforthepeople — CHANGES_REQUESTED** [review]
Left some comments inline

File: variables.tf (hunk 3 above)
Comment (SaiPrasannaGopularam) [reply variables.tf]: Can be given anything, however changing it to be an empty string to be safe

**Pull request merged**